### ISSUE TITLE
[question] any video results?

### ISSUE THREAD

**Issue by eisneim** [starboi-63/ArTEMIS#7]
thanks for sharing this great work! it would be nice to show us some comparison results

**Comment by starboi-63** [comment]
Hey there! I'm really sorry about the delay in my response (was very busy with some other work). I finally got around to adding more documentation for ArTEMIS and made a [project page](https://tanishmakadia.com/projects/artemis) for it on my website.

I did create a single demo output that I thought would be relatively challenging for the model - it's a chair I found with a pillow that has lots of stripes on it (once again, it's on the project page). I realize though that a single example output isn't very useful for comparison purposes. We were (and still are) students, so at the time we were paying for Google Colaboratory to get access to an A100 GPU to train the model over a few days. I now have access to compute through my university,  so I will try to make a few more in the future but can't guarantee how soon since I'm quite busy with school.

However, I can give quantitative results from running the model on the Vimeo-90k Septuplet Dataset's test set. Our problem formulation meant that 1 of 3 ground-truth frames was randomly selected (from timesteps 0.25, 0.5, and 0.75 - the central three frames for a single septuplet sample). The model was given this timestep value and tasked with recreating the frame. The average PSNR across all three of these timesteps on the test set was 33.029. The average SSIM was 0.95823. Finally, the average L1 loss was 0.012314. 

I know that other models that we took inspiration from (like [EDSC](https://arxiv.org/abs/2006.08070) and [DAIN](https://arxiv.org/abs/1904.00830) - shown below) achieved PSNR's of below 30 for the central three timesteps (granted they did not take in frames 2 and 6 as context).

<img width="400" alt="image" src="https://github.com/user-attachments/assets/cfa4c4ff-e43c-4995-9319-d00dd3fe6e28">

The comparison isn't one-to-one though, since our metrics are averages from randomly selecting a timestep for each sample whereas the metrics in the other papers are obtained by running the model on the entire test set for a single specific timestep.

I hope this helps a little at least - apologies again for not having more demo visuals.
